spikeinterface/core/recording_tools.py

```diff
@@ -229,18 +288,20 @@ def order_channels_by_depth(recording, channel_ids=None, dimensions=('x', 'y')):
     locations = locations[channel_inds, :]
 
     if isinstance(dimensions, str):
-        dim = ['x', 'y', 'z'].index(dimensions)
+        dim = ["x", "y", "z"].index(dimensions)
```

Comment:
@alejoe91 
Even it is unrelated to this PR, I think we have an error in this.
What we want by default is to short top to bottom and then left to right.
no ?
this lexsort do not do this because we should flip "x" otherwise we short from bottom to top and then left to right.
Am I wrong ?

Reply:
Lex sort goes backwards ;) https://numpy.org/doc/stable/reference/generated/numpy.lexsort.html

Reply:
Yup, it is the way we want, I was confused too and checked it